libraries/mediaviewer/api/src/main/kotlin/io/element/android/libraries/mediaviewer/api/local/LocalMediaViewState.kt

```diff
@@ -30,7 +30,10 @@ sealed interface PlayableState {
     data object NotPlayable : PlayableState
     data class Playable(
         val isPlaying: Boolean,
-        val isShowingControls: Boolean
+        val progressInMillis: Long,
```

Comment:
This is what we want to expose to the outer world, so I'm not sure this is the right place to add here
I'd have added all those values to the `MediaPlayerControllerState` instead and let the `Playable` derive some values from that

Reply:
62134022bbfa3939d3dd28a2aab77b57561f98d7